components/icons/README.md

```diff
@@ -1,3 +1,13 @@
-# Icons 
+# Icons
 
-To add an icon, place the .svg in this folder with and run `yarn icons`. The icon name will be taken from the filename of the .svg. 
+We use a subset of "Outlined" Material icons at Nulogy. To find and add a new one:
+
+1. Search https://material.io/resources/icons/ for the icon you need
+2. Download svg and rename to desired icon name
+3. Place svg in `/icons/` folder
+4. In the `/components` directory, run `yarn icons` to make the icon available to use
+5. Run `yarn start` and verify the icon was added correctly
+6. After verifying the icon is there, update storyshots with `yarn storyshots:update`
```

Comment:
specify where exactly to verify that the icon has been sucessfully added ex: visible in storybook under Icons